src/geometryRTheta/advection_field/advection_field_rtheta.hpp

```diff
@@ -311,15 +316,20 @@ class AdvectionFieldFinder
                         coord_rtheta_epsilon,
                         get_const_field(electrostatic_potential_coef));
 
-                // Gradient of phi in the physical index range (Cartesian index range)
-                double const deriv_x_phi_epsilon = deriv_r_phi_epsilon * inv_J_eps[0][0]
-                                                   + deriv_theta_phi_epsilon * inv_J_eps[1][0];
-                double const deriv_y_phi_epsilon = deriv_r_phi_epsilon * inv_J_eps[0][1]
-                                                   + deriv_theta_phi_epsilon * inv_J_eps[1][1];
+                // Gradient of phi in the physical domain (Cartesian domain)
+                // grad phi = (dr phi, dtheta phi) in covariant basis
+                double const grad_r_phi_epsilon = deriv_r_phi_epsilon;
+                double const grad_theta_phi_epsilon = deriv_theta_phi_epsilon;
```

Comment:
Why introduce an additional variable rather than just renaming `deriv_r_phi_epsilon` and `deriv_theta_phi_epsilon`?